### PR TITLE
[DPE-2331] Backport dashboards and alerting rules from vm to k8s

### PR DESCRIPTION
**Pull request by deusebio**

### CODE REVIEW

File: src/alert_rules/prometheus/zookeeper_metrics.rules [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+  - alert: Too Many Connections
+    expr: num_alive_connections{juju_charm!=".*"} > 100
+    for: 1m
+    labels:
+      severity: warning
+    annotations:
+      summary: "Instance {{ $labels.instance }} create too many connections"
+      description: "{{ $labels.instance }} of job {{$labels.job}} create too many connections: [{{ $value }}]."
+
+  - alert: Znode Total Cccupied Memory Too Big
```
Comment (marcoppenheimer) [inline src/alert_rules/prometheus/zookeeper_metrics.rules:59]: **nitpick:** I **think** it says `Ccc`. Probably should be `Occ`

**Review by marcoppenheimer — APPROVED** [review]
Looks fine. 

I think the `summary`s for the alert rules should be updated at some point, the grammar/participle is often incorrect and confusing.

E.g `Instance create too many znodes`. It's not clear if it should be past 'has created' or present 'creating'. 

A clearer summary might simply be 'High number of znodes on instance', or something to that effect.

No point blocking it in this PR, but we should backlog a rewording of these rules across these repos + k8s/vm.

**Review by zmraul — APPROVED** [review]

**Comment by deusebio** [comment]
@marcoppenheimer I have updated a bit the text for summary and descriptions. Please have a look, and feel free to propose other wording as you are the only native among us. I'd rather fix the working already in this PR, it does not seem a major thing that would require too much effort

File: src/alert_rules/prometheus/zookeeper_metrics.rules [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+      summary: "Instance {{ $labels.instance }} has too many pending sessions."
+      description: "Zookeeper has too many pending sessions and cannot keep up with client requests. This may be due to a overloaded cluster and/or sloppy performances."
+
+  - alert: Zookeeper Outstanding TLS Handshakes
+    expr: outstanding_tls_handshake{juju_charm!=".*"} > 0
+    for: 10m
+    labels:
+      severity: warning
+    annotations:
+      summary: "Instance {{ $labels.instance }} receives more TLS handshake than it can process."
```
Comment (marcoppenheimer) [inline src/alert_rules/prometheus/zookeeper_metrics.rules:154]: **nitpick:** receiving

**Review by marcoppenheimer — APPROVED** [review]
Left some very minor suggestions, but otherwise it's so much better than what we had before! Thanks for this!

**Pull request merged**